riposte-core/src/test/java/com/nike/riposte/server/handler/RoutingHandlerTest.java

```diff
@@ -182,4 +196,99 @@ public void findSingleEndpointForExecution_throws_MultipleMatchingEndpointsExcep
         // when
         handlerSpy.findSingleEndpointForExecution(requestInfoMock);
     }
+
+    @Test
+    public void doChannelRead_HttpRequest_throws_exception_when_content_length_header_greater_than_configured_global_request_limit() {
+        // given
+        doReturn(null).when(endpointMock).maxRequestSizeInBytesOverride();
+
+        maxRequestSizeInBytes = 10;
+        httpHeaders.set(CONTENT_LENGTH, 100);
+        httpHeaders.set(CONTENT_TYPE, APPLICATION_JSON);
```

Comment:
no, i'll remove them as they are not necessary for the testing of this functionality. i added them just for typical use case even though they are ignored.